src/dialog/reimu.c

```diff
@@ -734,67 +734,67 @@ DIALOG_TASK(reimu, StageExPreBoss) {
 	MSG(reimu, "Haven’t you ever read a book?");
 
 	FACE(yumemi, eyes_closed);
-	MSG(yumemi, "You mean fiction? I haven’t had a need for it in years.");
+	MSG(yumemi, "You mean fiction? Not in years.");
 	FACE(yumemi, sigh);
 	MSG(yumemi, "It’s a waste of time.");
 
 	FACE(reimu, unsettled);
 	MSG(reimu, "Wow, no wonder you’re so miserable.");
 
-	FACE(yumemi, sad);
-	MSG(yumemi, "Even as I introduced this technology to the world, nobody woke up from their dissociative haze. It was as it ever was.");
+	MSG(yumemi, "Do you see this place we're in, shrine maiden? Do you even know how it was made?");
+	MSG(yumemi, "Whether you believe it or not, this machine should have been able to save the world.");
 	FACE(yumemi, eyes_closed);
-	MSG(yumemi, "But at least, with this machine I’ve built, I can erase every so-called ‘Otherworld’ from the fabric of social reality.");
-	FACE(yumemi, normal);
+	MSG(yumemi, "But even when I revealed this technology to the world, nobody woke up from their dissociative haze.");
+	MSG(yumemi, "Nothing changed. And I figured out why.");
+	MSG(yumemi, "It's because of places like this. Why work on the real world when you can pretend to in another?");
 
 	FACE(reimu, assertive);
+	FACE(yumemi, normal);
 	MSG(reimu, "People won’t stop day-dreaming just because you tell them to. Fantasy is in people’s hearts and minds.");
-	FACE(reimu, sigh);
-	MSG(reimu, "So, even if you managed to destroy Gensōkyō—");
 
 	MSG(yumemi, "Yōkai, Gods, magicians, flying shrine maidens…");
 	FACE(yumemi, eyes_closed);
-	MSG(yumemi, "It’s all delusional nonsense.");
+	MSG(yumemi, "It’s all delusional nonsense. And I now have the power to erase it all.");
 
 	FACE(reimu, unsettled);
 	MSG(reimu, "So you want genocide?");
 
 	FACE(yumemi, surprised);
-	MSG(yumemi, "Genocide…? What a harsh word.");
+	MSG(yumemi, "Genocide…? What a nasty word.");
```

Comment:
Not sure why this had to change — I think "harsh" seems a bit more appropriate here?